crypto/bn/bn_exp.c

```diff
@@ -1041,30 +1039,45 @@ int BN_mod_exp_mont_consttime(BIGNUM *rr, const BIGNUM *a, const BIGNUM *p,
         }
 
         bits--;
```

Comment:
In such case code would much more readable if bits is not biased by 1.

Reply:
I tried to preserve as much of the original code as makes sense. But I agree that the original bits-- before the loop and the new bits+1 when calling bn_get_bits can be refactored out without losing anything. Will test and update the code accordingly.

Reply:
Updated.